src/main/java/com/celements/parents/DocumentParentsLister.java

```diff
@@ -0,0 +1,79 @@
+package com.celements.parents;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xwiki.component.annotation.Component;
+import org.xwiki.component.annotation.Requirement;
+import org.xwiki.model.reference.DocumentReference;
+
+import com.celements.web.service.XDocRecursionException;
+
+@Component
+public class DocumentParentsLister implements IDocumentParentsListerRole {
+
+  private static Logger _LOGGER = LoggerFactory.getLogger(DocumentParentsLister.class);
```

Comment:
We decided to add a leading underscore for private static fields quite a while ago. This is to make expected usage clear even though the field could be defined as package default to allow use in tests.